dashboard/src/actions/profileActions.js

```diff
@@ -62,32 +54,22 @@ export const sendForUpdate = () => async (dispatch, getState) => {
         ...updatedUserDetails,
       });
       if (response.status === 200) {
-        let toast = {
-          variant: "success",
-          title: "Updated!",
-        };
+        dispatch(constructToast("success", "Updated!"));
```

Comment:
What does it mean to invoke `dispatch()` on the result of `constructToast()` which itself invokes `dispatch()`?  (That is, why do we call `dispatch()` here _and_ inside `constructToast()`?  And...is the call inside `constructToast()` asynchronous?...in which case, what value are we passing to `dispatch()` here?)